woodwork/logical_types.py

```diff
@@ -830,6 +835,23 @@ def _regex_validate(regex_key, series, return_invalid_values):
             raise TypeValidationError(info)
 
 
+def _replace_nans(series):
```

Comment:
would be nice to add some type-hinting here jjust to start getting it more incorporated into WW